includes/class-wc-stripe-intent-controller.php

```diff
@@ -135,6 +136,88 @@ protected function handle_error( $e, $redirect_url ) {
 		wp_safe_redirect( $redirect_url );
 		exit;
 	}
+
+	/**
+	 * Creates a Setup Intent through AJAX while adding cards.
+	 */
+	public function create_setup_intent() {
+		if (
+			! is_user_logged_in()
+			|| ! isset( $_POST['stripe_source_id'] ) // ToDo: Make the parameter shorter.
```

Comment:
Does the todo here mean something like "change `stripe_source_id` to `source_id`"? FWIW I think `stripe_source_id` is perfectly fine 🙂

Reply:
That was the idea. Initially I called it `stripe_source_id` to avoid conflicts with other parameters, but now that I think about it, it actually makes more sense to leave it this way, as you said :)

I removed the comment in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1398/commits/a00f8345cc5fa1b2c2c1a0fbefb02fdb8c94a972